src/modules/sensors/vehicle_imu/VehicleIMU.cpp

```diff
@@ -326,6 +327,14 @@ void VehicleIMU::Run()
 			}
 
 			publish_status = true;
+
+			// start notifying the user periodically if there's continuous clipping
+			if ((hrt_elapsed_time(&accel.timestamp_sample) > 3_s)
```

Comment:
Shouldn't this be _last_clipping_notify?

Reply:
Only if we want it to actually work :smirk:.

Thanks @mhkabir